.travis/before_install.sh

```diff
@@ -57,17 +57,17 @@ cd ..
 git clone --depth=1 https://github.com/pulp/ansible-pulp.git
 if [ -n "$PULP_ROLES_PR_NUMBER" ]; then
   cd ansible-pulp
-  git fetch --depth=1 origin +refs/pull/$PULP_ROLES_PR_NUMBER/merge
-  git checkout FETCH_HEAD
+  git fetch --depth=1 origin pull/$PULP_ROLES_PR_NUMBER/head:$PULP_ROLES_PR_NUMBER
```

Comment:
I'm wondering what would happen if `PULP_ROLES_PR_NUMBER` is equal to an existing branch?

Reply:
Great question. The `--depth=1` on clone will only clone the master branch:

> --depth <depth>
> Create a shallow clone with a history truncated to the specified number of commits. Implies - -single-branch unless --no-single-branch is given to fetch the histories near the tips of all branches. If you want to clone submodules shallowly, also pass --shallow-submodules.

`$PULP_ROLES_PR_NUMBER` should be a number so I think we're good.